scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,90 @@
+import axios from 'axios';
+import _ from 'lodash';
+import { Task, TaskRunner } from './task';
+
+// const githubGrafanaUrl = 'https://github.com/grafana/grafana';
+
+interface CloseMilestoneOptions {
+  milestone: string;
+}
+
+const closeMilestoneTaskRunner: TaskRunner<CloseMilestoneOptions> = async ({ milestone }) => {
+  const gitHubUsername = process.env.GITHUB_USERNAME;
+  const gitHubToken = process.env.GITHUB_ACCESS_TOKEN;
+
+  if (!gitHubUsername || !gitHubToken) {
+    console.log('the close milestone operation needs a GITHUB_USERNAME and  GITHUB_ACCESS_TOKEN environment variables');
+    return;
+  }
+
+  const client = axios.create({
+    baseURL: 'https://api.github.com/repos/grafana/grafana',
+    timeout: 10000,
+    auth: {
+      username: gitHubUsername,
+      password: gitHubToken,
+    },
+  });
+
+  const cherryPickLabel = 'cherry-pick needed';
+
+  if (!/^\d+$/.test(milestone)) {
+    console.log('Use milestone number not title, find number in milestone url');
+    return;
+  }
+
+  const res = await client.get(`/milestones/${milestone}`, {});
+
+  const milestoneData = res.data;
```

Comment:
Assuming this variable is only providing context, why not just: `const milestoneState = res.data.state;`?